package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "grace-shopper-team-b",
+  "version": "1.0.0",
+  "description": "pnbkn made an update",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start:dev": "npm run webpack:dev & nodemon server.js --ignore assets/",
```

Comment:
But we should not assume anything about other peoples computers, we should install `nodemon` as a dev dependency if it is required to run this app as a developer, and then people will always get it with `npm i` and be able to use it by reaching into the binary files from `node_modules`

I mean: `./node_modules/.bin/nodemon server.js`